examples/knowledge_tuning/translation_example/flows/translate_flow_knowledge.yaml

```
@@ -0,0 +1,57 @@
+    source_lang: "kan_Knda"  # Source language (Kannada)
+    target_lang: "eng_Latn"  # Target language (English)
+    trans_model_id: ai4bharat/indictrans2-indic-en-dist-200M
+    client: http://localhost:8000/v1
+
+- block_type: LLMBlock
+  block_config:
+    block_name: question_generation
+    model_id: "ibm-granite/granite-3.3-2b-instruct"
+    config_path: configs/generate_question.yaml
+    output_cols:
+      - question
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)